src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/DetailsVideo.jsx

```diff
@@ -92,6 +92,52 @@ export default function DetailsVideo({ appointment, facilityData }) {
 }
 
 DetailsVideo.propTypes = {
-  appointment: PropTypes.object.isRequired,
-  facilityData: PropTypes.object,
+  appointment: PropTypes.shape({
```

Comment:
made Type checking more detailed for appt and facility props. Makes it easier to catch bugs with props...